packages/core/src/actor/connection.ts

```diff
@@ -20,6 +26,11 @@ export type ConnId = string;
 
 export type AnyConn = Conn<any, any, any, any, any, any, any>;
 
+export type ConnType =
```

Comment:
make a separate public union type. these consts are internal, and "gneericWebSocketConnection" is not something i want to use as a developer. you'll need to remap to cleaner names